src/main/resources/config.sample.yml

```diff
@@ -21,14 +23,15 @@ deployment:
 
 # the defined platforms to run
 platforms :
-# type to use
-    - type: mturk/Pybossa/dummy
```

Comment:
can you leave this in ? or at least write that mturk pybossa dummy are the only valid types?